brave/brave-core
# Fixes Brave Ads should not open the same target url for all creatives in a creative set

Resolves https://github.com/brave/brave-browser/issues/12653

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- See https://github.com/brave/brave-browser/issues/12653
- Confirm ads are served after upgrading from 1.16.x to 1.17.x
- Confirm ads are served after upgrading from 1.17.x to 1.18.x
- Confirm ads are served after upgrading from 1.18.x to 1.19.x

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
